tests/handlers/botas/test_botas_shipped_advisory.py

```diff
@@ -1213,7 +1211,7 @@ def test_bundle_include_previous_builds(self, get_original_build):
         mapping = self.handler._create_original_to_rebuilt_nvrs_map()
 
         self.assertEqual(get_original_build.call_count, 2)
-        self.assertEqual(mapping, expected_map)
+        self.assertDictEqual(mapping, expected_map)
```

Comment:
```
        get_original_build.side_effect = ["foo-1-100.1004", "bar-2-200.2007"]
        self.handler.event = BotasErrataShippedEvent("test_msg_id", self.botas_advisory)
        self.botas_advisory._builds = {
            "product_name": {
                "builds": [{"foo-1-100.1005": {"nvr": "foo-1-100.1005"}},
                           {"bar-2-200.2008": {"nvr": "bar-2-200.2008"}}]
            }
        }
        self.get_blocking_advisories.return_value = {"foo-3-300", "bar-2-300"}
        db_event = Event.get_or_create_from_event(db.session, self.handler.event)
        ArtifactBuild.create(
            db.session, db_event, "ed0", "image", original_nvr="foo-1-100.1003", rebuilt_nvr="foo-1-100.1004"
        )
        ArtifactBuild.create(
            db.session, db_event, "ed1", "image", original_nvr="foo-1-100.1002", rebuilt_nvr="foo-1-100.1003"
        )
        ArtifactBuild.create(
            db.session, db_event, "ed2", "image", original_nvr="foo-1-100", rebuilt_nvr="foo-1-100.1002"
        )
        ArtifactBuild.create(
            db.session, db_event, "ed3", "image", original_nvr="bar-2-200.2006", rebuilt_nvr="bar-2-200.2007"
        )
        ArtifactBuild.create(
            db.session, db_event, "ed4", "image", original_nvr="bar-2-200.2005", rebuilt_nvr="bar-2-200.2006"
        ),
        ArtifactBuild.create(
            db.session, db_event, "ed5", "image", original_nvr="bar-2-200", rebuilt_nvr="bar-2-200.2005"
        ),
        expected_map = {
            "bar-2-200": "bar-2-200.2008",
            "bar-2-200.2005": "bar-2-200.2008",
            "bar-2-200.2006": "bar-2-200.2008",
            "bar-2-200.2007": "bar-2-200.2008",
            "bar-2-300": "bar-2-200.2008",
            "foo-1-100": "foo-1-100.1005",
            "foo-1-100.1002": "foo-1-100.1005",
            "foo-1-100.1003": "foo-1-100.1005",
            "foo-1-100.1004": "foo-1-100.1005"
        }

        mapping = self.handler._create_original_to_rebuilt_nvrs_map()
        self.assertEqual(get_original_build.call_count, 2)
        self.assertDictEqual(mapping, expected_map)
```

Reply:
Thanks a lot, Qixiang! Now both the code and the test are much cleaner and understandable!